pyGHDL/cli/dom.py

```diff
@@ -316,7 +317,7 @@ def HandlePretty(self, args):
         for line in PP.formatDesign(self._design, 1):
             buffer.append(line)
 
-        print("\n".join(buffer))
+        logger.info("\n".join(buffer))
```

Comment:
This output is not printed by logger.

Reply:
As this is a test application, I propose reverting these back to print statements.